src/ztimem_object_name_shelp.fugr.z_timem_object_name_shelp.abap

```diff
@@ -0,0 +1,17 @@
+FUNCTION z_timem_object_name_shelp.
```

Comment:
Hi @larshp,

This says the Variable "z_timem_object_name_shelp" not used. But it is actually a Function Module. Is this a bug or a feature? :-)

It is true that it is still not used but it will be. I guess I shouldn't have committed it LOL.

Reply:
Now that I think about it, this FM is actually being used. As a search help user-exit. So I guess this is a bug, regardless of it being called variable.

I'll report it.